common/serial/serial.h

```diff
@@ -0,0 +1,219 @@
+//
+// Bareflank Hypervisor
+//
+// Copyright (C) 2015 Assured Information Security, Inc.
+// Author: Rian Quinn        <quinnr@ainfosec.com>
+// Author: Brendan Kerrigan  <kerriganb@ainfosec.com>
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+#ifndef SERIAL_PORT__H
+#define SERIAL_PORT__H
+
+#include <port_io.h>
+
+/** 
+ *  @file    serial.h
+ *  @author  Brendan Kerrigan
+ *  @date    2015/11/02  
+ *  @version 0.1 
+ *  
+ *  @brief Provide serial port access 
+ *
+ *  @section DESCRIPTION
+ *  
+ *  Provides a wrapper class around a serial port, which is
+ *  accessed via port I/O.
+ *  
+ *  Useful information:
+ *  http://wiki.osdev.org/Serial_Ports
+ *  http://www.sci.muni.cz/docs/pc/serport.txt
+ *
+ */
+
+#define COM1_IO_PORT 0x3f8
+#define COM2_IO_PORT 0x2f8
+#define COM3_IO_PORT 0x3e8
+#define COM4_IO_PORT 0x2e8
+
+/* Limits */
```

Comment:
Disagreed, perhaps the one liner one. Doesn't doxygen rely on C-style block comments?